source/guides/put-it-together/ddiscordbot.md

```diff
@@ -0,0 +1,303 @@
+Denizen and Discord: dDiscordBot
+--------------------------------
+
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
+
+### Introduction
+
+dDiscordBot is an **addon plugin** for Denizen that provides commands, script events, and tags for interacting with the Discord API. Basically, you can create a functioning Discord bot directly within Denizen!
+
+### What Can It Do?
+
+There are many libraries for various different programming languages for the Discord API. You might be familiar with some, such as [discord.js](https://discord.js.org), [discord.py](https://discordpy.readthedocs.io/en/stable/), and [JDA](https://github.com/DV8FromTheWorld/JDA). These libraries aim to cover **all aspects** of the API.
+
+Denizen is for Minecraft servers. Accordingly, dDiscordBot provides utilities that would be useful for a Minecraft server to interact with Discord. Nevertheless, some of the things you can do are:
+
+- Send, receive, and reply to messages
+  - Attach files, embeds, and buttons
+- Manage and reply to slash commands
+- Manage roles
+
```

Comment:
Just a mild suggestion from myself also, with it being directly connected to our Minecraft server, it may also be cool to mention that we can even run scripts and commands for our servers from Discord, that was a huge hype for me with Discord integration - only suggesting because you've only mentioned Discord features here it feels like